Lib/fontTools/feaLib/builder.py

```diff
@@ -298,13 +318,20 @@ def build_name(self):
             table.names = []
         for name in self.names_:
             nameID, platformID, platEncID, langID, string = name
+            # For featureNames block, nameID is 'feature tag'
+            # For cvParameters blocks, nameID is 'feature tag.block name'
```

Comment:
why do you have to use a dotted string as a dict key? Can't you use a tuple of strings?

Reply:
Perhaps. What would be the advantage, in your opinion?

Reply:
because you can (tuples of str are hashable, just like a str is) and because you avoid having to call `.split()` to get to one or the other item

Reply:
ok, I'll give it a try